lambdas/issue-credential/src/time-handler.ts

```diff
@@ -4,48 +4,56 @@ import { TimeEvent } from "./time-event";
 
 const logger = new Logger();
 
-const Unit = {
-  Seconds: "seconds",
-  Minutes: "minutes",
-  Hours: "hours",
-  Days: "days",
-  Months: "months",
-  Years: "years",
-  None: "none",
-};
+enum Unit {
+  Seconds = "seconds",
+  Minutes = "minutes",
+  Hours = "hours",
+  Days = "days",
+  Months = "months",
+  Years = "years",
+  None = "none",
+}
+
+export class TimeHandler implements LambdaInterface {
+  private readonly timeInSecs: number;
 
-const convert = (unit?: string): number => {
-  switch (unit) {
-    case Unit.Seconds:
-      return 1000;
-    case Unit.Minutes:
-      return 1000 * 60;
-    case Unit.Hours:
-      return 1000 * 60 * 60;
-    case Unit.Days:
-      return 1000 * 60 * 60 * 24;
-    case Unit.Months:
-      return 1000 * 60 * 60 * 24 * 30;
-    case Unit.Years:
-      return 1000 * 60 * 60 * 24 * 365;
-    case Unit.None:
-      return 1;
-    default:
-      throw new Error(`Unexpected time-to-live unit encountered: ${unit}`);
+  constructor() {
+    this.timeInSecs = this.msToSeconds(Date.now());
   }
-};
 
-const parseUnit = (value?: string): string => {
-  const unitKey = Object.keys(Unit).find(
-    (key) => Unit[key as keyof typeof Unit] === value?.toLowerCase()
-  );
-  if (unitKey) {
-    return Unit[unitKey as keyof typeof Unit];
+  private msToSeconds(ms: number): number {
+    return Math.floor(ms / 1000);
+  }
+
+  private convert(unit: Unit): number {
```

Comment:
```suggestion
  private convertToMs(unit: Unit): number {
```
Clearer name for the function, as it always converts to milliseconds

Reply:
Though `convertToMs` is still not accurate as the function doesn't actually convert anything, just returns a multipler to convert to ms, so perhaps it should be `toMsMultiplier`, but reallly it should probably take two args - a value (number) and the unit, and then just return the number of milliseconds.

Reply:
Originally that's what I did  see https://github.com/govuk-one-login/ipv-cri-toy-api/blob/287275436e3bf4a6aa8706467e36f2b228004c25/tests/utils/time-units.test.ts#L4

Reply:
This should be encapsulated in a class, as discussed on Slack